app/components/shared/Duration.js

```diff
@@ -1,20 +1,43 @@
+// @flow
+
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import 'intersection-observer';
+import VisibilityObserver from 'react-intersection-observer';
 import { second } from 'metrick/duration';
-import { getDurationString } from '../../lib/date';
-
-class Duration extends React.PureComponent {
+import moment from 'moment';
+import { getDuration, getDurationString } from '../../lib/date';
+
+type Props = {
+  from?: (moment$Moment | string | number | Date | number[]),
+  to?: (moment$Moment | string | number | Date | number[]),
```

Comment:
lol

Reply:
I’m pretty sure we technically only ever use the `string` approach, but mirroring the moment API underlying this seemed reasonable until I can audit it properly ;)